docs/auth/sexpr.md

```diff
@@ -0,0 +1,563 @@
+# S-expression based permission templates
+
+This is a proposal for an Auth service redesign based on defining
+permission templates to replace the current concept of 'roles'. The
+intent of these was to allow multiple permissions to be granted
+simultaneously (for example: MQTT subscribe and Cmdesc Rebirth), but the
+model is not flexible enough as all permissions in the role are applied
+to a single target. In addition, both MQTT and Cmdesc have implemented
+ad-hoc template expansions based on ConfigDB entries; these are
+generalised into a template language defined in S-expressions (LISP data
+structures) expressed as JSON arrays.
+
+## Data structure
+
+### Identity and ownership
+
+* Sparkplug Nodes are principals; their Sparkplug address information is
+  moved into the Auth service to sit alongside the Kerberos identity as
+  an alternative name.
```

Comment:
Can you elaborate on what this means? Does this contradict the UNS direction, should we choose to pursue it? With the UNS concept Sparkplug is only one of potentially many channels of data and we could expect other top-level namespaces (and their ingester pods) sending data into the `/UNS/v1` namespace.